pkg/kudoctl/cmd/install.go

```diff
@@ -19,15 +21,51 @@ var (
 		kubectl kudo install kafka --package-version=0`
 )
 
+// parseParameters parse raw parameter strings into a map of keys and values
+func parseParameters(raw []string, parameters map[string]string) error {
```

Comment:
at a min... it feels more idiomatic to have `func parseParameters(raw []string) (map[string]string, error) {`
and create the parameters in the function.

Reply:
hmm yeah, I also wonder why you decided to go this way 🤔 I thought it's because you pre-allocated it to some size but that does not seem true... So I would probably prefer what Ken is proposing too...

Reply:
You guys are good reviewers ;) I did it because otherwise calling the func with already declared `options.Parameters` is weird. You can't:
```
options.Parameters, err = parseParameters(parameters) // Nope
if err != nil {
}
```
but rather save it in a temporary `params` first:
```
params, err := parseParameters(parameters) // Yep
options.Parameters = params
if err != nil {
}
```

Admittedly, passing maps into functions is more c-style but if you feel strongly about it, I can change.

Reply:
Are we sure this doesn't work:  https://play.golang.org/p/BgLtqnwKR7k seems to work fine.  Not sure what's different about the playground example and what you're trying to do.

Reply:
Yeah, it also works with `var err error` (instead of `options.Parameters = params`). Seems that the majority is for it. I'll change